nemo/utils/import_guards.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+
+from nemo.utils import logging
+
+
+@contextmanager
```

Comment:
How would you check if import was successful or not ? We use a bool flag like HAS_APEX or NUMBA to be able to determine if the module could be imported or not

Reply:
Here I just wanted to suppress missing import. I use it only for annotation purposes ("Normalizer" is not a valid annotation if `Normalizer` is not imported!), but I don't care if it is missing (TTS model will fail with the appropriate message).
This doesn't sacrifice usability, but still will work in all cases.

If I need some library to be explicitly used, I thought about something like `@k2_required` decorators, which will allow to import everything-everywhere, but will raise an error if the method is called/ class is instantiated.

Reply:
`import_guard` is removed